src/ActivityProfile.js

```diff
@@ -1,8 +1,8 @@
 class ActivityProfile {
   constructor(user, data) {
-    this.user = user
-    this.data = data;
-    this.entries = data.filter(user => user.userID === this.user.id)
+    this.user = user;
+    this.entries = data.filter(user => user.userID === this.user.id);
+    this.friends = user.friends;
```

Comment:
Cool, we will need this to compare the users step count with their friends step count.